make release-alias-tag fails on prow image-pushing

```
Step #0: make[3]: Leaving directory '/workspace'
Step #0: make[2]: Leaving directory '/workspace'
Step #0: gcloud container images add-tag gcr.io/k8s-staging-cluster-api-nested/cluster-api-nested-controller:v20210608-ea129c6 gcr.io/k8s-staging-cluster-api-nested/cluster-api-nested-controller:main
Step #0: Created [gcr.io/k8s-staging-cluster-api-nested/cluster-api-nested-controller:main].
Step #0: Updated [gcr.io/k8s-staging-cluster-api-nested/cluster-api-nested-controller:v20210608-ea129c6].
Step #0: gcloud container images add-tag gcr.io/k8s-staging-cluster-api-nested/nested-controlplane-controller:v20210608-ea129c6 gcr.io/k8s-staging-cluster-api-nested/nested-controlplane-controller:main
Step #0: ERROR: Error during upload of: gcr.io/k8s-staging-cluster-api-nested/nested-controlplane-controller:main
Step #0: ERROR: (gcloud.container.images.add-tag) Not found: response: {'status': '404', 'content-length': '202', 'x-xss-protection': '0', 'transfer-encoding': 'chunked', 'server': 'Docker Registry', '-content-encoding': 'gzip', 'docker-distribution-api-version': 'registry/2.0', 'cache-control': 'private', 'date': 'Tue, 08 Jun 2021 18:57:01 GMT', 'x-frame-options': 'SAMEORIGIN', 'content-type': 'application/json'}
Step #0: Failed to fetch "v20210608-ea129c6" from request "/v2/k8s-staging-cluster-api-nested/nested-controlplane-controller/manifests/v20210608-ea129c6".: <no details provided>
Step #0: make[1]: *** [Makefile:353: release-alias-tag] Error 1
Step #0: make[1]: Leaving directory '/workspace'
Step #0: make: *** [Makefile:346: release-staging] Error 2

```

--- a/Makefile
+++ b/Makefile
@@ -266,6 +266,7 @@ docker-controlplane-build-%:
 .PHONY: docker-push-all ## Push all the architecture docker images
 docker-push-all: $(addprefix docker-infrastructure-push-,$(ALL_ARCH)) $(addprefix docker-controlplane-push-,$(ALL_ARCH))
 	$(MAKE) docker-push-core-manifest
+	$(MAKE) docker-push-nested-control-plane-manifest
 
 .PHONY: docker-push
 docker-push:
@@ -288,6 +289,15 @@ docker-push-core-manifest: ## Push the fat manifest docker image for the core im
 	docker manifest push --purge $(CONTROLLER_IMG):$(TAG)
 	$(MAKE) set-manifest-image MANIFEST_IMG=$(CONTROLLER_IMG) MANIFEST_TAG=$(TAG) TARGET_RESOURCE="./config/default/manager_image_patch.yaml"
 	$(MAKE) set-manifest-pull-policy TARGET_RESOURCE="./config/default/manager_pull_policy.yaml"
+
+.PHONY: docker-push-nested-control-plane-manifest
+docker-push-nested-control-plane-manifest: ## Push the fat manifest docker image for the nested control plane image.
+	## Minimum docker version 18.06.0 is required for creating and pushing manifest images.
+	docker manifest create --amend $(CONTROLPLANE_CONTROLLER_IMG):$(TAG) $(shell echo $(ALL_ARCH) | sed -e "s~[^ ]*~$(CONTROLPLANE_CONTROLLER_IMG)\-&:$(TAG)~g")
+	@for arch in $(ALL_ARCH); do docker manifest annotate --arch $${arch} ${CONTROLPLANE_CONTROLLER_IMG}:${TAG} ${CONTROLPLANE_CONTROLLER_IMG}-$${arch}:${TAG}; done
+	docker manifest push --purge $(CONTROLPLANE_CONTROLLER_IMG):$(TAG)
+	$(MAKE) set-manifest-image MANIFEST_IMG=$(CONTROLPLANE_CONTROLLER_IMG) MANIFEST_TAG=$(TAG) TARGET_RESOURCE="./controlplane/nested/config/default/manager_image_patch.yaml"
+	$(MAKE) set-manifest-pull-policy TARGET_RESOURCE="./controlplane/nested/config/default/manager_pull_policy.yaml"
 
 .PHONY: set-manifest-pull-policy
 set-manifest-pull-policy:

